libraries/Microsoft.Bot.Builder.AI.LUIS/Generator/Temperature.cs

```diff
@@ -13,7 +13,7 @@ public class Temperature : NumberWithUnits
         /// </summary>
         /// <param name="temperature">Temperature.</param>
         /// <param name="units">Units.</param>
-        public Temperature(double temperature, string units)
+        public Temperature(double? temperature, string units)
```

Comment:
All of these are breaking changes, which means we cannot take this code. We would need a *new* API instead.

Reply:
I guess I do see your point, but by the same token, it's broken as it is right now. Doubly-strange, considering that the base type uses nullable. Is there another way to move this forward? Perhaps "Temperature2" etc.?